src/main/kotlin/org/taktik/icure/services/external/rest/v1/controllers/be/KmehrController.kt

```diff
@@ -131,6 +132,7 @@ class KmehrController(
 					info.attachmentId,
 					null
 				)
+				)
 			}
 		} ?: throw IllegalArgumentException("Missing argument")
 	}
```

Comment:
you need to inject the coroutine context here

Reply:
I don't see how my implementation of generateDiaryNote differs from the existing generateSumehrV2